### ISSUE TITLE
Multimethods named with `**` are marked as earmuffed.

### ISSUE THREAD

**Issue by IGJoshua** [clj-kondo/clj-kondo#1896]
**version**: v2022.11.02

**platform**: Native clj-kondo from the AUR on Manjaro

**editor**: Emacs via clojure-lsp

**problem**:
Multimethods defined with a name that consists of one or more asterisks and nothing else (e.g. `*`, `**`, `***`, etc.) are marked as being non-dynamic earmuffed functions.

**repro**

```clojure
(ns repro)
(defmulti ** any?)
```

```sh
$ clj-kondo --lint repro.clj
repro.clj:2:11: warning: Var has earmuffed name but is not declared dynamic: **
linting took 24ms, errors: 0, warnings: 1
```

**config**

No config for the specified project, and no edits to user config.

**expected behavior**

The multimethod should not have a warning associated with it.

**Comment by borkdude** [comment]
Good catch :)
